index.html

```diff
@@ -1722,18 +1722,18 @@ <h6 data-lang="zh-hant">發表對本文檔的建議</h6>
             <p data-lang="en"><span class="leadin">Em dash and long dash.</span></p>
             <p data-lang="zh-hans" class="checkme"><span class="leadin">乙式括号与破折号</span></p>
             <p data-lang="zh-hant"><span class="leadin">乙式括號與破折號</span></p>
-            <p data-lang="en"><span class="uname">U+2E3A TWO-EM DASH</span> [⸺] or <span class="uname">U+2014 EM DASH</span> [—] should take one-character height and two-character widths, and long dash [——] can be used created by using two adjacent em dashes. </p>
+            <p data-lang="en"><span class="uname">U+2E3A TWO-EM DASH</span> [⸺] or <span class="uname">U+2014 EM DASH</span> [—] should take one character height and two character widths. Long dashes [——] can be created using two adjacent em dashes.</p>
             <p data-lang="zh-hans" class="checkme">乙式括号与破折号是占两个汉字空间的<span class="uname" translate="no">U+2014 EMDASH</span> [—(]或使用连续两个「——」)。</p>
             <p data-lang="zh-hant">乙式括號與破折號為佔二個漢字空間的<span class="uname">U+2E3A TWO-EM DASH</span> [⸺] 或<span class="uname">U+2014 EM DASH</span> [—]。</p>
           </li>
           <li>
             <p data-lang="en"><span class="leadin">Horizontal ellipsis.</span> </p>
             <p data-lang="zh-hans" class="checkme"><span class="leadin">删节号／省略号</span></p>
             <p data-lang="zh-hant"><span class="leadin">刪節號／省略号</span></p>
-            <p data-lang="en"><span class="uname">U+2026 HORIZONTAL ELLIPSIS</span> […] takes one-character height and two-characters in width. Two horizontal ellipses can be used together 「……] . </p>
+            <p data-lang="en"><span class="uname">U+2026 HORIZONTAL ELLIPSIS</span> […] takes one character height and two character widths. Two horizontal ellipses can be used together 「……] .</p>
```

Comment:
It's "one-character height and two-character widths" in the Em Dash section. I'm OK with either one, but I think it'd be good to be consistent.

Reply:
I'll check again for consistency.